### PR TITLE
TELCODOCS-867: 4.12 Ecosystems Release Notes

### PR DESCRIPTION
**Pull request by ogradyp**
https://issues.redhat.com/browse/TELCODOCS-867 Ecosystem Release Checklist (OCP 4.12 GA)

Applies to OpenShift version : 4.12

Preview: [Self Node Remediation Operator enhancements](https://54442--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-self-node-remediation-operator)
Preview: [Node Health Check Operator enhancements](https://54442--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-node-health-check-operator)
Preview: [OpenShift Container Platform on AWS Outposts](https://54442--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-aws-outposts)

https://54442--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-aws-outposts

Dev review to be complete/approved by @mshitrit @slintes @batzionb 
QE review to be complete/approved by @anna-savina @prabinovRedhat
Peer review complete/approved @jldohmann 

Thank you.

### CODE REVIEW

**Comment by batzionb** [comment]
lgtm

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -764,6 +764,23 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
...
 
+[id="ocp-4-12-self-node-remediation-operator"]
+==== Self Node Remediation Operator enhancements
+
+{product-title} now supports control plane fencing on the Self Node Remediation Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#control-plane-fencing-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes[Control Plane Fencing].
+
+[id="ocp-4-12-node-health-check-operator"]
+==== Node Health Check Operator enhancements
+
+{product-title} now supports control plane fencing on the Node Health Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#control-plane-fencing-node-health-check-operator_node-health-check-operator[Control Plane Fencing].
```
Comment (mshitrit) [inline release_notes/ocp-4-12-release-notes.adoc:775]: I suggest replacing  `Node Health Operator` with `Node Health Check Operator`

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -764,6 +764,23 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 
 {product-title} now supports the crun container runtime in Technology Preview. You can switch between the crun container runtime and the default container runtime as needed by using a `ContainerRuntimeConfig` custom resource (CR). For more information, see xref:../nodes/containers/nodes-containers-using.adoc#nodes-containers-runtimes[About the container engine and container runtime].
 
+[id="ocp-4-12-self-node-remediation-operator"]
+==== Self Node Remediation Operator enhancements
+
+{product-title} now supports control plane fencing on the Self Node Remediation Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#control-plane-fencing-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes[Control Plane Fencing].
```
Comment (anna-savina) [inline release_notes/ocp-4-12-release-notes.adoc:770]:
```suggestion
{product-title} now supports control plane fencing by the Self Node Remediation Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#control-plane-fencing-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes[Control Plane Fencing].
```

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (ogradyp) [reply release_notes/ocp-4-12-release-notes.adoc]: > I suggest replacing `Node Health Operator` with `Node Health Check Operator`

Thanks @mshitrit Good catch! I've applied the update.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (ogradyp) [reply release_notes/ocp-4-12-release-notes.adoc]: > 

Thanks @anna-savina  I've applied the update.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 3]
```diff
@@ -764,6 +764,23 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
...
+[id="ocp-4-12-node-health-check-operator"]
+==== Node Health Check Operator enhancements
+
+{product-title} now supports control plane fencing on the Node Health Check Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#control-plane-fencing-node-health-check-operator_node-health-check-operator[Control Plane Fencing].
+
+The Node Health Check Operator now also includes a web console plugin for managing Node Health Checks. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#eco-node-health-check-operator-creating-node-health-check_node-health-check-operator[Creating a node health check].
+
+For installing or updating to the latest version of the Node Health Check Operator, use the `stable` subscription channel. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#installing-node-health-check-operator-using-cli_node-health-check-operator[Installing the Node Health Check Operator by using the CLI].
+
+[id="ocp-4-11-logging"]
```
Comment (slintes) [inline release_notes/ocp-4-12-release-notes.adoc:781]: should this belong into this PR?

**Review by slintes — COMMENTED** [review]
One remark about something which looks wrong for this PR. Otherwise lgtm

File: release_notes/ocp-4-12-release-notes.adoc (hunk 3 above)
Comment (ogradyp) [reply release_notes/ocp-4-12-release-notes.adoc:781]: > should this belong into this PR?

Thanks @slintes . No, lines 780/781 are not part of this PR. Maybe they got added my mistake. They point to 4.11 Logging, and are probably a placeholder for others to add that content. My content is immediately above this in lines 767-779 inclusive.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 4]
```diff
@@ -965,6 +891,23 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
...
+
+{product-title} now supports control plane fencing by the Self Node Remediation Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-self-node-remediation-operator.adoc#control-plane-fencing-self-node-remediation-operator_self-node-remediation-operator-remediate-nodes[Control Plane Fencing].
+
+[id="ocp-4-12-node-health-check-operator"]
+==== Node Health Check Operator enhancements
+
+{product-title} now supports control plane fencing on the Node Health Check Operator. In the event of node failure, you can follow remediation strategies on both worker nodes and control plane nodes. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#control-plane-fencing-node-health-check-operator_node-health-check-operator[Control Plane Fencing].
+
+The Node Health Check Operator now also includes a web console plugin for managing Node Health Checks. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#eco-node-health-check-operator-creating-node-health-check_node-health-check-operator[Creating a node health check].
```
Comment (jldohmann) [inline release_notes/ocp-4-12-release-notes.adoc:904]: ISG nit:
```suggestion
The Node Health Check Operator now also includes a web console plug-in for managing Node Health Checks. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#eco-node-health-check-operator-creating-node-health-check_node-health-check-operator[Creating a node health check].
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 5]
```diff
@@ -143,83 +143,9 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
...
-
-For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#root-device-hints_preparing-to-install-with-agent-based-installer[About root device hints].
-
-[id="ocp-4-12-agent-based-install-dhcp"]
-==== Agent-based installation supports DHCP
-With the Agent-based {product-title} Installer, you can deploy to environments where you rely on DHCP to configure networking for all the nodes, as long as you know the IP that at least one of the systems will receive. This IP is required so that all nodes use it as a meeting point.
-For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#dhcp[DHCP].
+[id="ocp-4-12-aws-outposts"]
+==== {product-title} on AWS Outposts
+{product-title} 4.12 is now supported on the AWS Outposts platform. With AWS Outposts you can deploy edge-based worker nodes, while using AWS Regions for the control plane nodes. The documentation for this feature is currently unavailable and is targeted for release at a later date.
```
Comment (jldohmann) [inline release_notes/ocp-4-12-release-notes.adoc:148]:
```suggestion
{product-title} {product-version} is now supported on the AWS Outposts platform. With AWS Outposts you can deploy edge-based worker nodes, while using AWS Regions for the control plane nodes. The documentation for this feature is currently unavailable and is targeted for release at a later date.
```

**Review by jldohmann — COMMENTED** [review]
Looking good! Left a couple of minor comments below

File: release_notes/ocp-4-12-release-notes.adoc (hunk 4 above)
Comment (ogradyp) [reply release_notes/ocp-4-12-release-notes.adoc]: Thanks @jldohmann I've applied your feedback.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 5 above)
Comment (ogradyp) [reply release_notes/ocp-4-12-release-notes.adoc]: Thanks @jldohmann I've applied your feedback.

**Comment by mburke5678** [comment]
@ogradyp PR needs a squash

**Comment by ogradyp** [comment]
Thank you @mburke5678. Much appreciated.

**Pull request merged**